ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -384,45 +384,72 @@ if get_option('enable-openvino')
 endif
 
 if get_option('enable-mediapipe')
-
-  cmd = run_command('sh', '-c', 'echo $MEDIAPIPE_HOME')
-  MEDIAPIPE_HOME = cmd.stdout().strip()
-  message('['+MEDIAPIPE_HOME+']')
-
-  mediapipe_env_exist = run_command('[', '-z', MEDIAPIPE_HOME, ']').returncode()
-  mediapipe_dir_exist = run_command('[', '-d', MEDIAPIPE_HOME, ']').returncode()
-
-  if mediapipe_env_exist == 1 and mediapipe_dir_exist == 0
-    message('MEDIAPIPE_HOME: ' + MEDIAPIPE_HOME)
-    mediapipe_incdir = include_directories(
-      MEDIAPIPE_HOME,
-      join_paths(MEDIAPIPE_HOME, 'bazel-bin'),
-      join_paths(MEDIAPIPE_HOME, 'bazel-mediapipe/external/eigen_archive'),
-      join_paths(MEDIAPIPE_HOME, 'bazel-mediapipe/external/com_google_absl'),
-    )
-
-    cmd = run_command('sh', '-c', meson.source_root() + '/tools/development/gen_mediapipe_libs.sh ' + nnstreamer_libdir)
-
-    mediapipe_internal_dep = cxx.find_library('mediapipe_internal', dirs: nnstreamer_libdir)
-    mediapipe_external_dep = cxx.find_library('mediapipe_external', dirs: nnstreamer_libdir)
-    
-    nnstreamer_filter_mediapipe_deps = [glib_dep, gst_dep, nnstreamer_dep, mediapipe_external_dep, mediapipe_internal_dep]
-
-    filter_sub_mp_sources = ['tensor_filter_mediapipe.c']
-
-    nnstreamer_filter_mediapipe_sources = []
-    foreach s : filter_sub_mp_sources
-      nnstreamer_filter_mediapipe_sources += join_paths(meson.current_source_dir(), s)
-    endforeach
-    
-    shared_library('nnstreamer_filter_mediapipe',
-      nnstreamer_filter_mediapipe_sources,
-      dependencies: nnstreamer_filter_mediapipe_deps,
-      include_directories: mediapipe_incdir,
-      install: true,
-      install_dir: filter_subplugin_install_dir
-    )
+  message('host system: ' + host_machine.system())
+  message('host cpu family: ' + host_machine.cpu_family())
+
+  if host_machine.system() == 'linux' and host_machine.cpu_family() == 'x86_64'
+    cmd = run_command('sh', '-c', 'echo $MEDIAPIPE_HOME')
+    MEDIAPIPE_HOME = cmd.stdout().strip()
+
+    mediapipe_env_exist = run_command('[', '-z', MEDIAPIPE_HOME, ']').returncode()
+    mediapipe_dir_exist = run_command('[', '-d', MEDIAPIPE_HOME, ']').returncode()
+
+    if mediapipe_env_exist == 1 and mediapipe_dir_exist == 0
```

Comment:
Same here. Check if there is an error and exit if it's an error. Don't nest the code blocks of "no-error" case.